Translation of Available methods section (#94)
wlsf82/protractor-helper: README_PT-BR.md

  - [Exemplo utilizando o protractor-helper](#exemplo-utilizando-o-protractor-helper)
- [Instalação](#instalação)
- [Como usar e exemplos](#como-usar-e-exemplos)
- [Helpers disponíveis](#helpers-disponíveis)

  <details>

>     <p><summary>Abrir para visualizar os helpers disponíveis</summary>

Reviewer: Que tal assim?

```suggestion
    <p><summary>Abrir para visualizar todas as funções disponíveis</summary>
```

[Aqui você encontrará exemplos de uso de cada um dos métodos disponíveis.](docs/EXAMPLES.md#examples) (Importante: leia as notas no topo)

> ## Helpers disponíveis

Reviewer: ```suggestion
## Funções disponíveis
```

> Abaixo segue a lista de todos os helpers disponíveis no protractor-helper com seus respectivos exemplos:

Reviewer: ```suggestion
Abaixo segue a lista de todas as funções disponíveis no protractor-helper com seus respectivos exemplos:
```

### `setTimeout`

> Este método permite alterar o tempo de timeout de todos os métodos chamados após `setTimeout`.

Reviewer: ```suggestion
Este método permite alterar o _timeout_ de todos os métodos chamados após o `setTimeout`.
```

> O `timeoutInMilliseconds` padrão é 5000 millisegundos

Reviewer: ```suggestion
O `timeoutInMilliseconds` padrão é 5000 milisegundos
```

> Caso seja invocado sem que um parametro seja passado, o timeout será definido como o padrão.

Reviewer: ```suggestion
Caso seja invocado sem que um parametro seja passado, o _timeout_ será definido como o padrão.
```

> > Um exemplo de uso desse método é a necessidade de assegurar que a tela seja carregada dentro de XX segundos, devido uma necessidade expressa pelo gerente de produtos. Isso ocorre frequentemente em sistemas de e-commerce emdatas especiais como a Blackfriday.

Reviewer: ```suggestion
> Um exemplo de uso desse método é a necessidade de assegurar que a tela seja carregada dentro de XX segundos, devido uma necessidade expressa pelo gerente de produtos. Isso ocorre frequentemente em sistemas de _e-commerce_ em datas especiais como a Black Friday.
```

[Exemplo](docs/EXAMPLES.md#settimeout)

### `isCurrentUrlDifferentFromBaseUrl`

> Este método retorna um valor booleano dependendo se a url atual é diferente da url base. Não necessita de parâmetros.

Reviewer: ```suggestion
Este método retorna um valor booleano dependendo se a url atual é diferente da url base e não necessita de parâmetros.
```

[Exemplo](docs/EXAMPLES.md#iscurrenturldifferentfrombaseurl)

### `waitForElementPresence`

Este método espera até que um elemento esteja presente no DOM.

> Nota: Um elemento pode estar presente mas não visível. Se você deseja esperar até que o elemento esteja visível, use o método `waitForElementVisibility`

[Exemplo](docs/EXAMPLES.md#waitforelementpresence)

### `waitForElementNotToBePresent`

Este método é o oposto do anterior, de tal forma que espera até que um elemento não esteja presente no DOM.

[Exemplo](docs/EXAMPLES.md#waitforelementnottobepresent)

### `waitForElementVisibility`

> Este método aguarda até que um elemento esteja visível na página. Estar visível não significa apenas estar presente no DOM, mas também tem altura e largura superior a 0.

Reviewer: ```suggestion
Este método aguarda até que um elemento esteja visível na página. Estar visível significa não apenas estar presente no DOM, mas também ter altura e largura maior que 0.
```

[Exemplo](docs/EXAMPLES.md#waitforelementvisibility)

### `waitForElementNotToBeVisible`

Este método é o oposto do anterior, de tal forma que espera que um elemento não esteja visível na página. Por não visível, entenda-se que o elemento pode estar no DOM mas não estar visível.

[Exemplo](docs/EXAMPLES.md#waitforelementnottobevisible)

### `click`

> Este método é utilizado para efetuar um clique em um elemento assim que estiver em um estado viável e habilitado para o clique.

Reviewer: ```suggestion
Este método é utilizado para efetuar um clique em um elemento assim que estiver em um visível e habilitado para o clique.
```

Author: Nesse acredito que a palavra "estado" foi removida indevidamente

Reviewer: Oops.

Reviewer: Vou ajustar a sugestão.

Reviewer: Done.

[Exemplo](docs/EXAMPLES.md#click)

### `hoverAndClick`

Este método é utilizado para mover o mouse sobre o elemento assim que estiver presente no DOM e então efetuar um clique.

[Exemplo](docs/EXAMPLES.md#hoverAndClick)

### `fillFieldWithText`

Este médoto preenche um campo de input com um texto assim que estiver visível.

[Exemplo](docs/EXAMPLES.md#fillFieldWithText)

### `uploadFileIntoInputField`

Este método efetua o upload de um arquivo em um input de arquivos assim que estiver presente no DOM.

[Exemplo](docs/EXAMPLES.md#uploadFileIntoInputField)

### `clear`

Este método limpa o conteúdo de um campo de input assim que estiver visível.

[Exemplo](docs/EXAMPLES.md#clear)

### `clearFieldAndFillItWithText`

Este método limpa o conteúdo de um campo de input assim que estiver visível, então preenche o mesmo com um texto.

[Exemplo](docs/EXAMPLES.md#clearFieldAndFillItWithText)

### `tap`

> Este método executa  a ação de toque em um elemento HTML capaz de eventos de toque/clique assim que estiver em um estado viável de toque/clique. É utilizado quando se está fazendo testes web mobile em emuladores, por exemplo.

Reviewer: ```suggestion
Este método executa  a ação de toque em um elemento HTML capaz de eventos de toque/clique assim que estiver em um estado possível de toque/clique. Tal método é utilizado quando se está fazendo testes web _mobile_ em emuladores, por exemplo.
```

[Exemplo](docs/EXAMPLES.md#tap)

### `waitForTextToBePresentInElement`

Este método espera por um texto específico em um determinado elemento HTML.

[Exemplo](docs/EXAMPLES.md#waitfortexttobepresentinelement)

### `waitForTextNotToBePresentInElement`

> Este método é o oposto do anterior, de tal forma que espera que um texto não esteja presente em um determinado elemento do HTML.

Reviewer: ```suggestion
Este método é o oposto do anterior, de tal forma que espera que um texto não esteja presente em um determinado elemento HTML.
```

[Exemplo](docs/EXAMPLES.md#waitfortextnottobepresentinelement)

### `waitForUrlToBeEqualToExpectedUrl`

Este método aguarda até que a URL seja igual a outra esperada. Este método é útil quando se quer executar ações em elementos apenas se estiver na URL correta.

[Exemplo](docs/EXAMPLES.md#waitforurltobeequaltoexpectedurl)

### `waitForUrlNotToBeEqualToExpectedUrl`

> Este método aguarda até que a URL não seja igual a uma a uma determinada URL. Este método é útil quando se quer continuar a executar ações em elementos apenas quando não estiver em uma URL específica.

Reviewer: ```suggestion
Este método aguarda até que a URL não seja igual a uma a uma outra determinada URL. Este método é útil quando se quer continuar a executar ações em elementos apenas quando não estiver em uma URL específica.
```

[Exemplo](docs/EXAMPLES.md#waitforurlnottobeequaltoexpectedurl)

### `waitForUrlToContainString`

Este método aguarda até que a URL contenha um determinado texto. É utilizado quando se deseja fazer verificações baseadas na URL atual.

[Exemplo](docs/EXAMPLES.md#waitforurltocontainstring)

### `waitForUrlNotToContainString`

Este método aguarda até que a URL não contenha um determinado texto. É utilizado quando se deseja fazer verificações baseadas na URL atual.

[Exemplo](docs/EXAMPLES.md#waitforurlnottocontainstring)

### `fillFieldWithTextAndPressEnter`

> Este método preenche um campo de input com um texto assim que estiver visível e então simula a ação de pressionar a tecla ENTER do teclado. É utilizado em casos como fazer uma busca e pressionar ENTER, em vez de preencher o campo e selecionar o botão de busca, por exemplo.

Reviewer: ```suggestion
Este método preenche um campo de input com um texto assim que estiver visível e então simula a ação de pressionar a tecla ENTER do teclado. É utilizado em casos como fazer uma busca e pressionar ENTER, em vez de preencher o campo e clicar no botão de busca, por exemplo.
```

[Exemplo](docs/EXAMPLES.md#fillFieldWithTextAndPressEnter)

### `scrollToElement`

> Este método é usado para rolar a página até um elemento até que esteja visível no DOM.

Reviewer: ```suggestion
Este método é usado para rolar a página até um elemento até que tal elemento esteja visível no DOM.
```
